jest.config.js

```diff
@@ -1,14 +1,16 @@
+const { jsWithBabelESM: tsjPreset } = require('ts-jest/presets')
 const customJestConfig = {
   preset: 'ts-jest',
   moduleDirectories: ['node_modules', '<rootDir>/'],
   testEnvironment: 'jest-environment-jsdom',
   transform: {
+    ...tsjPreset.transform,
```

Comment:
What kind of black-magicery is this? :D

Reply:
It's from the ts-jest documentation: https://kulshekhar.github.io/ts-jest/docs/getting-started/presets/